tests/formulagrader/test_formulagrader.py

```diff
@@ -466,7 +468,7 @@ def test_fg_config_expect():
 
     # If trying to use comparer, a detailed validation error is raised
     expect = ("to have 3 arguments, instead it has 2 for dictionary value @ "
-              "data\['answers'\]\[0\]\['expect'\]\['comparer'\]")
+              "data\['answers'\]\[0\]\[u?'expect'\]\['comparer'\]")
```

Comment:
This I like :-)

Reply:
Yeah, I think I put a note in the commit for the debug log, but I thought that replacing the debug log test with a regexp match would involve much too much escaping... {u} isn't too terrible.